test/plugins/scenarios/go-redisv9/config/excepted.yml

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+  - serviceName: go-redisv9
+    segmentSize: ge 1
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: redis/set
+            parentSpanId: 0
+            spanId: 1
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:6379
+            skipAnalysis: false
+            tags:
+              - {key: cache.type, value: redis}
+              - {key: cache.args, value: not null}
```

Comment:
`cache.op` tag seems not being added.

Please refer to virtual cache docs, https://skywalking.apache.org/docs/main/next/en/setup/service-agent/virtual-cache/

Those tags are important if you expect all telemetry data is available on the dashboard.

The catalog we are using at Java side could be found here

https://github.com/apache/skywalking-java/blob/6621f82739340bbaaac2338cb06ccbe3d38513a5/apm-sniffer/apm-sdk-plugin/jedis-plugins/jedis-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jedis/v4/JedisPluginConfig.java

Reply:
Tags are added accordingly.